modules/release-notes/pages/index.adoc

```diff
@@ -169,6 +169,13 @@ AdminPortal UI for the xref:#finer_grain_query_privileges[fine-grained query pri
 * Fixed regression problem with installing queries which create lists containing mixed types of numeric data (GLE-7928).
 * Fixed int64 value underflow error by explicitly type casting uint64 (CORE-4108).
 * Restored the ability to run the TigerGraph `gcollect` command on Kubernetes (TP-6351).
+* Improved the performance of GSQL queries containing delete statements intended for deleting all vertices of a given type (GLE-8931).
+* Fixed issue where the loading job would hang when loading a blank file with only a header line (TP-6635).
+* Fixed issue where the job status was not correctly reported when the loading job failed to start (TP-6131).
+* Fixed issue with query installation failure for single-node queries which initialize vertex set variables in conditional branches (GLE-8846).
+* Added the 'graph' field to CDC messages generated by the TigerGraph CDC service (CORE-4146).
+* Fixed issue with error codes in the log when a CDC message failed to deliver to external Kafka (CORE-4326).
+* Fixed critical disk issue caused by the rebuilder getting stuck in a partitioned cluster after dropping vertex or edge attributes (CORE-4357).
 
```

Comment:
```suggestion
* Fixed issue where local accumulators defined across multiple lines in a query were misinterpreted as a file in the GSQL client (GLE-8260).
* Added validation to prevent EXE from reading files with negative length and enforced gRPC message maximum size when the length was too large (TP-6764).
```